app/models/in_progress_form.rb

```diff
@@ -19,8 +20,25 @@ def cast(value)
   before_save :serialize_form_data
   before_save :set_expires_at
 
+  scope :for_user, lambda { |user|
```

Comment:
👍nice way to scope it to both ids

Reply:
@patrickvinograd would there ever be a case where `user.uuid` is NULL?

Reply:
@ericbuckley Yes, I think we're headed for a case where we get an inbound SSOe session that will not have an idme_uuid. Do you think the query needs updating for that case?

Reply:
I think so, if we get a user with a NULL uuid, this query will return all users that also have a NULL uuid plus the one use with the matching account_id.

Reply:
But there's not a separate uuid column on the InProgressForm model - we're overloading the user_uuid column to always either be an idme_uuid or an account_id (prefixed with 'acct:'). The column still has a non-null constraint. 
In other words I think the difference is that you needed that logic in the Account model because you added a new secid column and then had to look up across two columns either of which might be null, but that's not the case here. 
But I should add a test case for nil user.uuid.

Reply:
And maybe a guard that account_id and user.uuid are not both nil.

Reply:
that makes sense, if `user_uuid` is never NULL then ignore my last comment

Reply:
I can't easily add such a spec for nil uuid because it's validated to be non-nil, but I'll add some more careful handling of nil account_ids